lib/takeaway.rb

```diff
@@ -0,0 +1,65 @@
+require_relative 'customer.rb'
+require_relative 'order.rb'
+require_relative 'menu.rb'
+require_relative 'send_text.rb'
+
+class Takeaway
+  def initialize(menu = Menu.new, order_class = Order, text = SendText.new)
```

Comment:
Really nice of extraction of methods into other classes so that your takeaway class worked well and clearly in IRB